compiler/luci/pass/include/luci/Pass/FuseBCQPass.h

```diff
@@ -26,11 +26,12 @@ namespace luci
  * @brief  Class to fuse certain pattern of subgraph into CircleBCQFullyConnected or CircleBCQGather
  *
  */
-struct FuseBCQPass final : public logo::Pass
+struct FuseBCQPass final : public luci::Pass
 {
   const char *name(void) const final { return "luci::FuseBCQPass"; }
 
-  bool run(loco::Graph *g) final;
+  bool run(luci::Module *m) final;
+  bool run(loco::Graph *) final;
```

Comment:
Doesn't seem to be for this PR

Reply:
This PR is for changing the `run(loco::Graph)` function to return false and it does not use the parameter.
As a result it caused compile error if I do not remove the unused parameter.

Reply:
Really?

Reply:
Yes, that's why I added last commit.
(http://nnfw.mooo.com:8080/job/nnfw/job/master/job/pr-nncc-debug/4911/)